### PR TITLE
Continuous integration

### LINKED ISSUE #31: Create CI test system

### PR DESCRIPTION
**Pull request by natanlao**
Closes #31. 

This PR sets up continuous integration for this repo with Travis using `treon` and also tweaks some notebooks to help tests pass.

If I've messed something up with `.travis.yml`, we probably won't see it until this PR is merged into master. In that case, I would have to follow up with another PR.

~~On merge, tests will fail. This is because most of the notebooks are broken - referring to search queries that have no results, files that can't be found, etc. I've fixed the login notebook (which appeared to be out of date anyway) but the rest of the fixes seem to be out of the scope of this PR.~~ Per stand up, all tests are now skipped and we'll re-add them as they pass.

I could have implemented the same testing using `nbconvert` instead of Treon, but using Treon was too easy.

### CODE REVIEW

File: test/requirements-dev.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+treon==0.1.2
```
Comment (DailyDreaming) [inline test/requirements-dev.txt]: Should add: `-r requirements.txt` to this line in order to include those dependencies as well.  Example:
https://github.com/HumanCellAtlas/data-store/blob/master/requirements-dev.txt.in#L14

File: tasks/Log In/.ipynb_checkpoints/Log In-checkpoint.ipynb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Things fall apart when logging in\n",
+    "First, assume the HCA CLI is already installed.\n",
+    "\n",
+    "The docs say to log in with `hca dss login`. This never returns if your run it within a notebook:"
```
[the remark this answers is not in the document]
Comment (natanlao) [reply tasks/Log In/.ipynb_checkpoints/Log In-checkpoint.ipynb]: Ah, my bad, I didn't proofread the rest of the notebook...

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+python:
+- 3.7
+
+install:
+- pip install -r test/requirements.txt
+- pip install -r test/requirements-dev.txt
+
+before_script:
+# To ignore a directory, we can just delete the directory
+# so that treon doesn't see it.
```
[the remark this answers is not in the document]
Comment (natanlao) [reply .travis.yml]: 👃

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+language: python
+
+python:
+- 3.7
```
[the remark this answers is not in the document]
Comment (natanlao) [reply .travis.yml]: Fixed!

**Pull request merged**